### PR TITLE
refactor(common): update error messages of NgOptimizedImage directive

### PR DESCRIPTION
**Pull request by AndrewKushnir**
This commit updates error messages that the NgOptimizedImage directive throws or logs into a console.

## PR Type
What kind of change does this PR introduce?

- [x] Refactoring (no functional changes, no api changes)

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by AndrewKushnir** [comment]
cc @[khempenius

**Comment by AndrewKushnir** [comment]
cc @khempenius

**Review by pkozlowski-opensource — COMMENTED** [review]
The change LGTM overall but the CI is consistently (?) failing - I've restarted it and see the same error. Could you please have a look? Maybe we need to rebase?

File: packages/common/src/directives/ng_optimized_image/image_loaders/image_loader.ts [hunk 1]
```diff
@@ -73,16 +73,15 @@ function throwInvalidPathError(path: unknown, exampleUrls: string[]): never {
   throw new RuntimeError(
       RuntimeErrorCode.INVALID_LOADER_ARGUMENTS,
       `Image loader has detected an invalid path. ` +
```
Comment (khempenius) [inline packages/common/src/directives/ng_optimized_image/image_loaders/image_loader.ts:75]: Mention the invalid path - i.e. "Image loader has detected an invalid path (${path})."

File: packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts [hunk 2]
```diff
@@ -316,14 +314,14 @@ function assertNoConflictingSrcset(dir: NgOptimizedImage) {
...
-            `at a time that does not disrupt lazy loading.`);
+        `${imgDirectiveDetails(dir.rawSrc)} both \`srcset\` and \`rawSrcset\` have been set. ` +
+            `Supplying both of these attributes is not necessary and will break lazy loading. ` +
+            `The NgOptimizedImage directive will set \`srcset\` itself based on the value of ` +
+            `\`rawSrcset\`. To fix this, please remove the \`srcset\` attribute.`);
   }
 }
 
-// Verifies that the `rawSrc` is not a Base64-encoded image.
+// Verifies that the ` rawSrc` is not a Base64-encoded image.
```
Comment (khempenius) [inline packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts:324]:
```suggestion
// Verifies that the `rawSrc` is not a Base64-encoded image.
```

File: packages/common/src/directives/ng_optimized_image/image_loaders/image_loader.ts [hunk 3]
```diff
@@ -73,16 +73,15 @@ function throwInvalidPathError(path: unknown, exampleUrls: string[]): never {
...
-          `Expecting a path matching one of the following formats: ${exampleUrlsMsg}` +
-          ` - but got: \`${path}\``);
+          `To fix this, supply a path using one of the following formats: ${exampleUrlsMsg}`);
 }
 
 function throwUnexpectedAbsoluteUrlError(path: string, url: string): never {
   throw new RuntimeError(
       RuntimeErrorCode.INVALID_LOADER_ARGUMENTS,
       `Image loader has detected an absolute URL used for one of the \`rawSrc\` ` +
```
Comment (khempenius) [inline packages/common/src/directives/ng_optimized_image/image_loaders/image_loader.ts:82]: Sneaking in one more change...

Image loader has detected an  tag with an invalid `rawSrc` attribute: ${url}. This image loader expects `rawSrc` to be a relative URL - however the provided value is an absolute URL. To fix this, provide `rawSrc` as a relative URL.

**Review by khempenius — APPROVED** [review]

**Pull request merged**